primefaces/src/main/resources/META-INF/resources/primefaces/staticmessage/staticmessage.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * __PrimeFaces StaticMessage Widget__
+ * 
+ * StaticMessage widget to handle events.
+ *
+ * @interface {PrimeFaces.widget.StaticMessageCfg} cfg The configuration for the {@link StaticMessage| StaticMessage widget}.
+ * You can access this configuration via {@link PrimeFaces.widget.BaseWidget.cfg|BaseWidget.cfg}. Please note that this
+ * configuration is usually meant to be read-only and should not be modified.
+ * @extends {PrimeFaces.widget.BaseWidgetCfg} cfg
+ */
+PrimeFaces.widget.StaticMessage = PrimeFaces.widget.BaseWidget.extend({
+
+    /**
+     * @override
+     * @inheritdoc
+     * @param {PrimeFaces.PartialWidgetCfg<TCfg>} cfg
+     */
+    init: function(cfg) {
+        this._super(cfg);
+        this.bindEvents();
+    },
+
```

Comment:
We probably need ARIA attributes on the Close Icon letting the screen reader know its a close button?  role="button" and AriaLabel = close?   We should check other close buttons like in Sidebar?

Reply:
```js
        if(this.cfg.showCloseIcon) {
            this.closeIcon.attr('role', 'button');
            this.closeIcon.attr('aria-label', PrimeFaces.getAriaLabel('close'));
        }
```

Is what sidebar.js does in the client side

Reply:
Hmm.. too bad that we have one file for front end and one for back end. Would be great if we had one file (backend messages) and transformed that to front end JSON by splitting keys on `.` or something.

Reply:
We have moved 99% of ARIA label stuff to the UI client side where it belongs.  It makes translations easier and makes more sense to translate "on the glass" instead of half in the backend and half in the frontend.